Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -367,6 +368,18 @@ ResourceLoaderOptions loaderOptions(
     setBufferedRanges(WTFMove(intersectionRanges));
 }
 
+void MediaPlayerPrivateWebM::updateDurationFromTrackBuffers()
+{
+    MediaTime highestEndTime = MediaTime::zeroTime();
+    for (auto& trackBuffer : m_trackBufferMap.values()) {
+        if (!trackBuffer.get().lastDecodeTimestamp())
```

Comment:
this should be presentationTimestamp()
even though in practice they are the same.

decodeTimestamp is not something that should ever impact anything user facing.